CorsixTH/Lua/audio.lua

```diff
@@ -100,8 +100,11 @@ function Audio:init()
     - Uses titles from MIDI.TXT if found, else the filename.
   --]]
   local midi_txt -- File name of midi.txt file, if any.
-  local waveform = { MP3=true, OGG=true, WAV=true, AIFF=true, VOC=true, FLAC=true }
-  local instructional = { MID=true, MOD=true, XM=true, XMI=true }
+  local waveform = list_to_set({"OGG", "OPUS", "FLAC", "WV", "WAV", "WAVE",
```

Comment:
Are you sure they dropped VOC?

Reply:
The codec is in 2.8, but it's not in the music.c detection.

I can't get voc files, or the xmi from CD or demo TH to play in my build or Schnapple's, the xmi error "Unrecognized file type (not AIFF or AIFC)" seems like it's detecting a different magic word. Is it successful on Windows or Linux?

Reply:
For VOC:
```bash
Lua\audio.lua:583: Could not load music file 'C:\temp\audio\sample3.voc' (ModPlug_Load failed)
stack traceback:
        [C]: in function 'coroutine.yield'
        Lua\app.lua:1102: in function <Lua\app.lua:1097>
```

XMI (non TH directory)
```bash
Lua\audio.lua:583: Could not load music file 'C:\temp\audio\CANDY.XMI' (Unrecognized file type (not AIFF or AIFC))
stack traceback:
        [C]: in function 'coroutine.yield'
        Lua\app.lua:1102: in function <Lua\app.lua:1097>
```